bpf/aya-bpf-macros/src/expand.rs

```diff
@@ -238,10 +238,8 @@ impl CgroupSkb {
             } else {
                 format!("cgroup_skb/{}", attach)
             }
-        } else if let Some(name) = &self.name {
-            format!("cgroup/skb/{}", name)
         } else {
-            ("cgroup/skb").to_owned()
+            return Err(Error::new_spanned(&self.expected_attach_type, "missing attach type"));
```

Comment:
This is not correct. It's actually fine for the section to not explicitly declare the attach type.
Please back out these changes.